integration/nemo/examples/supervised_fine_tuning/README.md

```diff
@@ -157,12 +157,25 @@ The curves shown are:
 - yellow: oasst1 
 - green: dolly
 - blue: alpaca
-- magenta: three datasets combined, 'centralized' training
+- magenta: three datasets combined
 
 The light curves are for single site training, and the dark curves are for FedAvg
 
-As shown, the global model from FedAvg is able to generate a loss comparable to that of centralized training.
+As shown, the global model from FedAvg is able to generate a loss comparable to that of training with all three datasets combined.
 
+Loss curve is one indication of model performance, but we also want to see if the model is able to generate reasonable text. Hence, benchmarked the trained models with standard language modeling tasks under 0-shot setting, including HellaSwag(H), PIQA(P), and WinoGrande(W). 
```

Comment:
"Hence, benchmarked" -> "Hence, we benchmarked"
"0-shot" -> "zero-shot".

Also, can you add links to the zeros shot task datasets?

Reply:
Thanks! Added